dist/ramda.js

```diff
@@ -3263,6 +3246,35 @@
 
     /**
      * Converts an object into an array of key, value arrays.
+     * Only the object's own properties are used.
+     * Note that the order of the output array is not guaranteed to be
+     * consistent across different JS platforms.
+     *
+     * @func
+     * @memberOf R
+     * @category Object
+     * @sig {String: *} -> [[String,*]]
+     * @param {Object} obj The object to extract from
+     * @return {Array} An array of key, value arrays from the object's own properties.
+     * @example
+     *
+     *      R.toPairs({a: 1, b: 2, c: 3}); //=> [['a', 1], ['b', 2], ['c', 3]]
+     */
+    var toPairs = _curry1(function toPairs(obj) {
+        var pairs = [];
+        for (var prop in obj) {
+            if (Object.prototype.hasOwnProperty.call(obj, prop)) {
```

Comment:
If that was the original reason for including `_has`, we're no longer fulfilling the objective. Here's the [current implementation][1]:

```javascript
module.exports = function _has(prop, obj) {
    return Object.prototype.hasOwnProperty.call(obj, prop);
};
```

I don't agree with the original reasoning. If the global environment is damaged, all bets are off.


[1]: https://github.com/ramda/ramda/blob/e29cf515e0714189856927617ec22c7e12667633/src/internal/_has.js